site/src/components/Helmet.js

```diff
@@ -36,45 +27,28 @@ const Helmet = ({ theme = {} }) => (
       return (
         <ReactHelmet htmlAttributes={{ lang: 'en' }}>
           <meta charSet="utf-8" />
-          <title>{title}</title>
+          <title>{`${title} - ${description}`}</title>
           <meta name="description" content={description} />
-          <link rel="shortcut icon" href={`https:${profile.favicon32.src}`} />
+          <link rel="shortcut icon" href={logo32} />
           <meta name="theme-color" content={theme.background} />
-          <meta name="image" content={`https:${profile.favicon32.src}`} />
+          <meta name="image" content={logo32} />
           <meta itemProp="name" content={title} />
           <meta itemProp="description" content={description} />
-          <meta itemProp="image" content={`https:${profile.favicon32.src}`} />
+          <meta itemProp="image" content={logo32} />
           <meta name="og:title" content={title} />
           <meta name="og:description" content={description} />
-          <meta name="og:image" content={`https:${profile.bigIcon.src}`} />
+          <meta name="og:image" content={logo512} />
```

Comment:
Please check if this image is working 🤔 

https://www.heymeta.com/url/deploy-preview-172--reach4help.netlify.com/

Reply:
Looks like it's working now :)